config.md

```diff
@@ -1475,4 +1475,189 @@ Ustawienie nie działa na linie już wcześniej wyświetlone.
 
 Dostępne wartości:
 * `true` - włączone
-* `false` - wyłączone
\ No newline at end of file
+* `false` - wyłączone
+
+
+## `scripts.ui.auto_wrap_main_window.enabled`
+
+Długość linii (miejsce jej łamania dokładniej mówiąc) w głównym oknie będzie zależna od jego szerokości.
+Ustawienie szczególnie użyteczne dla osób przerzucających okno na rózne wielkościowo monitory. 
+Ustawienie nie działa na linie już wcześniej wyświetlone.
+
+Dostępne wartości:
+* `true` - włączone
+* `false` - wyłączone
+
+## `scripts.character.state_reporting.<cecha>.<wartosc_liczbowa>`
```

Comment:
Czy idea jest ze maja byc to konfigurowalne wartosci? Jesli tak, to nie ma tego w configu